server/src/controllers/queryController.js

```diff
@@ -0,0 +1,26 @@
+const Query = require("../models/queryModel");
+const Response = require("../helpers/response");
```

Comment:
A very beautiful work, well-done @James-nyagatare,  however for your code to look much better you should always leave a space on top of your class and also space between two methods.